content/extraProblems.json

```diff
@@ -602,6 +602,18 @@
       "solutionMetadata": {
         "kind": "internal"
       }
+    },
+    {
+      "uniqueId": "usaco-966",
+      "name": "MooBuzz",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=966",
+      "source": "Silver",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Binary Search"],
```

Comment:
>       "tags": ["Binary Search"],

:thinking: isn't your solution O(1)?

Reply:
the problem could also be done with binary search. i couldn't find an appropriate tag for my solution